### DISCUSSION TITLE
增加不依赖播放键的推荐页刷新

### DISCUSSION THREAD

**Discussion by aobozhou** [Paladinfeng/MiaoProject#188] [Ideas]
应该有不少用户和我一样是使用电视自带的遥控器控制Apple TV的。但因为电视遥控器不像Apple TV 遥控器一样有单独的播放键，所以没有办法刷新推荐页。可否增加一个在UI中的按钮来支持点击刷新？或者把“推荐”Tab设置成第二下点击刷新？

**Comment by Paladinfeng** [comment]
还是建议开启CEC功能之后，用Apple TV的遥控器，刷新不只有几个Tab页，还有一些内部的列表页
目前还没想到更好的解决方案
例如 NF / D+ / YouTube 之类的，都是没有手动刷新功能的

**Reply by glm9412** [reply]
是否能像云视听小电视一样，点击tab，刷新当前tab页面内容？
